### PR TITLE
Added comment for cache structure change in multiselect

### PR DESCRIPTION
**Pull request by meghaarora42**
https://github.com/sferik/rails_admin/pull/2412
https://github.com/sferik/rails_admin/issues/2231
https://code.google.com/p/v8/issues/detail?id=164

### CODE REVIEW

**Comment by meghaarora42** [comment]
@mshibuya Anything missing?

File: app/assets/javascripts/rails_admin/ra.filtering-multiselect.js [hunk 1]
```diff
@@ -189,6 +189,14 @@
...
     },
 
+      // Cache key is stored in the format `o_<option value>` to avoid JS
+      // engine coercing string keys to int keys, and thereby preserving
+      // the insertion order. The value for each key is in turn an object
+      // that stores the option tag's HTML text and the value. Example:
+      // cache = {
+      //   'o_271': { id: 271, value: 'CartItem #271'},
+      //   'o_270': { id: 270, value: 'CartItem #270}'
+      // }
```
Comment (mshibuya) [inline app/assets/javascripts/rails_admin/ra.filtering-multiselect.js]: Please keep comment style consistent with another comment in this file.

File: app/assets/javascripts/rails_admin/ra.filtering-multiselect.js [hunk 2]
```diff
@@ -189,6 +189,14 @@
       });
     },
 
+      // Cache key is stored in the format `o_<option value>` to avoid JS
+      // engine coercing string keys to int keys, and thereby preserving
+      // the insertion order. The value for each key is in turn an object
+      // that stores the option tag's HTML text and the value. Example:
+      // cache = {
+      //   'o_271': { id: 271, value: 'CartItem #271'},
+      //   'o_270': { id: 270, value: 'CartItem #270}'
```
Comment (mshibuya) [inline app/assets/javascripts/rails_admin/ra.filtering-multiselect.js]: Syntax is wrong :cry:

**Comment by mshibuya** [comment]
Thanks:star2:

**Pull request merged**